app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -172,6 +173,15 @@ class AddProfileActivityTest {
     assertThat(label).isEqualTo(context.getString(R.string.add_profile_activity_label))
   }
 
+  @Test
+  fun testAddProfileActivity_hasCorrectRequiredString() {
+    activityTestRule.launchActivity(createAddProfileActivityIntent())
+    val label = activityTestRule.activity.add_profile_activity_required_heading_text_view.text
+
+    // Verify that the activity shows the correct *Required string for add_profile_activity_required_heading_text_view
+    assertThat(label).isEqualTo(context.getString(R.string.add_profile_required))
```

Comment:
This test case has been written incorrectly. Please check below test case in this file `testAddProfileActivity_inputName_opensProfileChooserActivity ` and implement a test similar to that.